pybuildkite/builds.py

```diff
@@ -291,6 +294,10 @@ def rebuild_build(self, organization, pipeline, build_number):
             + rebuild
         )
 
+    @staticmethod
+    def __process_meta_data(meta_data):
```

Comment:
This method will throw an error if meta_data is `None`. Perhaps we should only call it if meta_data exists. I also think meta_data should be defaulted to be an empty dictionary so it's more clear to the user what type to use.

Reply:
Nice catch thanks! I decided to treat `None` inside this function so it is consistent with the `__get_build_states_query_param(states)` function which is also always called and just returns None if the input is not set:

https://github.com/pyasi/pybuildkite/blob/be821c2f1cac04ea622b72d322f1ee57d937a597/pybuildkite/builds.py#L318-L328